src/main/java/com/soboleoko/medicalclinic/controller/VisitController.java

```diff
@@ -0,0 +1,35 @@
+package com.soboleoko.medicalclinic.controller;
+
+import com.soboleoko.medicalclinic.mapper.VisitMapper;
+import com.soboleoko.medicalclinic.model.VisitDTO;
+import com.soboleoko.medicalclinic.service.VisitService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RestController
+@RequestMapping("/visits")
+@RequiredArgsConstructor
+public class VisitController {
+    private final VisitService visitService;
+    private final VisitMapper visitMapper;
+
+    @PostMapping("/create")
```

Comment:
Raczej nie użwyamy w sciezkach czasownikow. Metoda HTTP okresla nam jaką wykonujemy operację w tym przypadku mamy POST więc utworzenie to po co jeszcze dodawac create